grr/server/grr_response_server/flows/general/osquery.py

```diff
@@ -12,16 +12,14 @@
 from grr_response_server import flow_responses
 
 
-TRUNCATED_ROWS_COUNT = 10
+TRUNCATED_ROW_COUNT = 10
 
-def _TruncateRows(table: rdf_osquery.OsqueryTable) -> rdf_osquery.OsqueryTable:
-  result = rdf_osquery.OsqueryTable()
-
-  result.query = table.query
-  result.header = table.header
-  result.rows = table.rows[:TRUNCATED_ROWS_COUNT]
-
-  return result
+def _GetTotalRowCount(
+  responses: flow_responses.Responses[rdf_osquery.OsqueryResult],
+) -> int:
+  project_length = lambda cur_response: len(cur_response.table.rows)
+  row_lengths = map(project_length, responses)
+  return sum(row_lengths)
```

Comment:
Initially, I read `project_length` as "length of a project" rather than "make a projection of length". I would rename it to `row_count` (or `table_row_count`) and then remove `row_lengths` variable.

Also, no need for the `cur_` prefix, in the lambda parameter (since there is no "previous", or "next" or anything alike to make it more meaningful).

```python
  row_count = lambda response: response.table.rows
  return sum(map(row_count, responses))
```